draft-ietf-httpbis-cache-latest.xml

```diff
@@ -705,7 +705,7 @@
 </t>
 <t>
    Clients can send the max-age or min-fresh request directives (<xref
-   target="cache-request-directive" />) to constrain or relax freshness
+   target="cache-request-directive" />) to attempt to constrain or relax freshness
```

Comment:
'to attempt to' doesn't read well. Can we restructure this?

Reply:
I'll try a rewording.